apps/ui/components/application/funnel/form.ts

```diff
@@ -500,13 +500,32 @@ export function createDefaultPage1Section(
   };
 }
 
-export function convertApplicationPage3(app?: Maybe<ApplicantFieldsFragment>): ApplicationPage3FormValues {
+function isOrganisationEmpty(app: ApplicantFieldsFragment): boolean {
+  return (
+    app.organisationName === "" &&
+    app.organisationIdentifier === "" &&
+    app.organisationCoreBusiness === "" &&
+    app.organisationStreetAddress === "" &&
+    app.organisationCity === "" &&
+    app.organisationPostCode === ""
+  );
+}
+
+export function convertApplicationPage3(app: Maybe<ApplicantFieldsFragment>): ApplicationPage3FormValues {
   const isOrganisation = app?.applicantType !== ReserveeType.Individual;
 
   const hasBillingAddress =
     app?.applicantType === ReserveeType.Individual ||
     (app?.billingStreetAddress != null && app?.billingStreetAddress !== "");
 
+  // complex due to we want to default this to false even for Nonprofits unless the user
+  // is editing an existing application (they have once set this explicitly to true)
+  const reserveeIsUnregisteredAssociation =
+    app != null &&
+    app.applicantType === ReserveeType.Nonprofit &&
+    !isOrganisationEmpty(app) &&
+    app.organisationIdentifier === "";
```

Comment:
Double negative makes this check unnecessarily complicated, I'd change the function to something like `isAnyOrganisationFieldFilled` to make this easier to understand